dashboard/app/views/layouts/_header.html.haml

```diff
@@ -13,12 +13,12 @@
         |
       = link_to t('landing.report_bug'), @script_level.report_bug_url(request), target: '_blank'
 
-  .navbar-static-top.header{class: ('hide_on_mobile' if true)}
+  .navbar-static-top.header{class: 'hide_on_mobile'}
     .container{style: (@full_width ? 'width: 880px;' : '')}
       .navbar-header
         .row.header_row
           .header_logo
-            = link_to(image_tag('codeorg-studio-logo.png'), '/', style: 'background-color: transparent;')
+            = link_to(image_tag('codeorg-studio-logo.png'), CDO.code_org_url('/'))
```

Comment:
hm... did you intend to change the header logo to go to code.org instead of studio.code.org?